2.7/s2i/bin/assemble

```diff
@@ -8,6 +8,14 @@ function should_collectstatic() {
   is_django_installed && [[ -z "$DISABLE_COLLECTSTATIC" ]]
 }
 
+function virtualenv_bin() {
+  if [[ $(head /etc/redhat-release) =~ ^Fedora.* ]]; then
```

Comment:
iirc there was some issue with `[[ ... ]]` not being available on macs due to older bash. Can we use grep here instead?